catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlResult.java

```diff
@@ -163,6 +164,16 @@ private String getDisplayName(TransformerDescriptors descriptors, String id, Str
     return title.replaceFirst("^Export( as)?\\s+\\b", "");
   }
 
+  private static boolean isBlacklistedTransformer(TransformerDescriptors descriptors, String id) {
+    for (String transformer : descriptors.getBlackListedMetacardTransformers()) {
+      if (id.endsWith(transformer)) {
```

Comment:
Why ends with instead of `.equals()`?

Reply:
The IDs of the actions are all prepended with `ddf.catalog.transformer`